.github/workflows/ci-cd.yml

```
@@ -90,7 +90,7 @@ jobs:
         uv run make mototest
     - name: Upload coverage to Codecov
       if: ${{ matrix.upload-coverage }}
-      uses: codecov/codecov-action@v5.4.2
+      uses: codecov/codecov-action@ad3126e916f78f00edff4ed0317cf185271ccc2d
```

If you keep the abbreviated tags in comments, Dependabot will keep them in sync with SHA hashes:
```suggestion
      uses: codecov/codecov-action@ad3126e916f78f00edff4ed0317cf185271ccc2d  # v5.4.2
```
(here and in other places)